lib/projects/project_member.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.Projects.ProjectMember do
+  @moduledoc """
+  Defines a Member of a Project.
+  """
+
+  alias GlassFactoryApi.Projects.ProjectMember
+
+  @type t() :: %ProjectMember{
+          id: String.t(),
+          user_id: String.t(),
+          project_id: String.t()
+        }
+
+  defstruct [
+    :id,
+    :user_id,
+    :project_id
+  ]
+
+  @doc """
+  Parser a map into a ProjectMember struct.
+
+  ## Examples
+
+      iex> GlassFactoryApi.Projects.ProjectMember.to_struct(%{"id" => 1, "user_id" => 123, "project_id" => 456})
+      %ProjectMember{
+        id: "1",
+        user_id: "123",
+        project_id: "456"
+      }
+  """
+  @spec to_struct(map()) :: ProjectMember.t()
+  def to_struct(attrs) do
+    %ProjectMember{
```

Comment:
Most style guides recommend `__MODULE__` to self-reference the current module:

- https://github.com/christopheradams/elixir_style_guide#module-pseudo-variable
- https://github.com/rrrene/elixir-style-guide#reference-current-module
- https://github.com/lexmag/elixir-style-guide#current-module-reference